command/agent/local.go

```diff
@@ -499,6 +516,8 @@ func (l *localState) syncChanges() error {
 		if err := l.syncNodeInfo(); err != nil {
 			return err
 		}
+	} else {
+		l.logger.Printf("[DEBUG] agent: Node info in sync")
```

Comment:
This one is low-rate and not really printed in response to traffic so I figured it was `DEBUG`-worthy.